src/eval/execute.rs

```diff
@@ -178,6 +186,20 @@ fn generate_result_from_response(resp: Response, channel: Channel, is_private: b
     }
 }
 
+async fn paste_to_pb(client: &Client, code: &str) -> reqwest::Result<String> {
+    let resp = client
+        .post("https://paste.mozilla.org/api/")
+        .form(&[
+            ("lexer", "rust"),
+            ("expires", "2073600"),
+            ("content", code),
+        ])
+        .header("content-length", 0)
+        .send().await?
+        .text().await?;
+    Ok(resp.trim_matches('"').to_string())
```

Comment:
I don't think it is reliable. How about pasting it as JSON?

<img width="729" alt="image" src="https://github.com/upsuper/telegram-rustevalbot/assets/28441561/513d7085-5d59-46b3-b6c0-d9cc5c1d97bd">

Reply:
mozilla.paste.org gives an `"..url.."`
may I should check their documents

Reply:
here: https://docs.dpaste.org/api/#form-parameters

By the way, `"..url.."` is also a valid JSON; therefore, do `.json::<String>()` can simplify a lot of works.